client/Command/ListSnapshots/SnappersData.h

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright (c) [2019] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of version 2 of the GNU General Public License as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact Novell, Inc.
+ *
+ * To contact Novell about this file by physical or electronic mail, you may
+ * find current contact information at www.novell.com.
+ */
+
+#ifndef SNAPPER_CLI_COMMAND_LIST_SNAPSHOTS_SNAPPERS_DATA_H
+#define SNAPPER_CLI_COMMAND_LIST_SNAPSHOTS_SNAPPERS_DATA_H
+
+#include <string>
+#include <vector>
+#include <map>
+
+#include "client/Command/ListSnapshots.h"
+#include "client/proxy.h"
+
+namespace snapper
+{
+    namespace cli
+    {
+
+	class Command::ListSnapshots::SnappersData
+	{
+
+	public:
+
+	    class Table;
+
+	    class Csv;
+
+	    class Json;
+
+	    enum Attribute {
+		ID, TYPE, PRE, DATE, USER, USED_SPACE, CLEANUP, DESCRIPTION, USERDATA, PRE_ID, PRE_DATE,
+		POST_ID, POST_DATE, CONFIG, SUBVOLUME
```

Comment:
Any reason to use "id" instead of "num"? That thing is always called num so far.

Reply:
Also the difference between PRE and PRE_ID is not clear.